roles/kubernetes/control-plane/tasks/kubeadm-setup.yml

```diff
@@ -1,18 +1,4 @@
 ---
-- name: kubeadm | Check if old apiserver cert exists on host
-  stat:
-    path: "{{ kube_cert_dir }}/apiserver.pem"
-    get_attributes: no
-    get_checksum: no
-    get_mime: no
-  register: old_apiserver_cert
-  delegate_to: "{{ groups['kube-master'] | first }}"
-  run_once: true
-
-- name: kubeadm | Migrate old certs if necessary
-  import_tasks: kubeadm-migrate-certs.yml
-  when: old_apiserver_cert.stat.exists
```

Comment:
oh wow this is ancient code I wrote related to non-kubeadm to kubeadm migration

Reply:
yeah when `git blame` gives 2017/2018 I know I'm on something :)